### PR TITLE
feat: add missing translation for Korean

### PR DESCRIPTION
**Pull request by zipizigi**
add new message translation for korean

### CODE REVIEW

File: priv/gettext/ko/LC_MESSAGES/default.po [hunk 1]
```diff
@@ -638,19 +638,19 @@ msgstr "공기압"
...
 #, elixir-autogen, elixir-format
 msgid "Dog Mode"
-msgstr ""
+msgstr "애견 모드"
 
 #: lib/teslamate_web/live/car_live/summary.ex:139
 #, elixir-autogen, elixir-format
 msgid "Dog mode is enabled"
-msgstr ""
+msgstr "애견 모드가 켜져있습니다.
```
Comment (JakobLichterfeld) [inline priv/gettext/ko/LC_MESSAGES/default.po:651]: this needs to be in `"` atz the end as well

**Review by JakobLichterfeld — CHANGES_REQUESTED** [review]
Thanks for your change. One small finding, see inline comment

File: priv/gettext/ko/LC_MESSAGES/default.po (hunk 1 above)
Comment (zipizigi) [reply priv/gettext/ko/LC_MESSAGES/default.po]: Oops, I made a mistake.
problem has been fixed.

**Review by JakobLichterfeld — APPROVED** [review]
lgtm

**Pull request merged**